plugins.txt

```diff
@@ -1,21 +1,22 @@
 docker-workflow:1.9.1
 ansible:0.6.2
-pipeline-milestone-step:1.2
-envinject:2.1.3
-aws-credentials:1.16
-credentials-binding:1.10
-ws-cleanup:0.32
-jobConfigHistory:2.15
+envinject:2.1.4
+aws-credentials:1.23
+credentials-binding:1.13
+ws-cleanup:0.34
+jobConfigHistory:2.18
 copyartifact:1.38.1
-s3:0.10.10
-ssh-agent:1.13
-extended-choice-parameter:0.75
+s3:0.10.12
+bouncycastle-api:2.16.2
```

Comment:
I've added this here, because otherwise an older version seems to be there, and the `ssh-agent` plugin can't load properly.